l1-contracts/deploy-scripts/DecentralizeGovernanceUpgradeScript.s.sol

```diff
@@ -1,4 +1,5 @@
 // SPDX-License-Identifier: MIT
```

Comment:
`ProxyAdmin` and `ITransparentUpgradeableProxy` imports are not updated to use the v4 alias:
```
import {ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
import {ITransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
```
It should be changed to:
```
import {ProxyAdmin} from "@openzeppelin/contracts-v4/proxy/transparent/ProxyAdmin.sol";
import {ITransparentUpgradeableProxy} from "@openzeppelin/contracts-v4/proxy/transparent/TransparentUpgradeableProxy.sol";
```